tests/test_snapshots/test_chain_snapshots.py

```diff
@@ -0,0 +1,277 @@
+from collections import Counter
+from decimal import Decimal
+
+import pytest
+import responses
+from pytest import approx
+
+from badger_api.requests import badger_api
+from helpers.constants import BOOST_CHAINS
+from helpers.enums import BalanceType, Network
+from rewards.snapshot.chain_snapshot import (
+    chain_snapshot,
+    chain_snapshot_usd,
+    parse_sett_balances,
+    sett_snapshot,
+)
+
+BADGER_TOKEN_ADDR = "0x19D97D8fA813EE2f51aD4B4e04EA08bAf4DFfC28"
```

Comment:
This is the bBADGER token address, not BADGER.  Correct address for the token is `0x3472A5A71965499acd81997a54BBA8D852C6E53d`.  We can also use the `constants.py` file to get the tokens in a more clean way

Reply:
Fixed